docs/DeveloperGuide.md

```diff
@@ -320,6 +320,10 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Glossary
 
+* **Medical Administrator**: A person who oversees, plan, direct, and coordinate health care delivery.
+* **Patients**: A person receiving or registered to receive medical treatment.
+* **Nurses**: A person trained to care for the sick or infirm, especially in a hospital.
+* **HealthCare Xpress**: A desktop app for managing patients that require home-visits.
```

Comment:
Maybe the 'c' here should be consistent. should be small 'c' I guess? cuz the readme.md there is small 'c'.